src/_pytest/pytester.py

```diff
@@ -669,6 +669,11 @@ def copy_example(self, name=None):
         else:
             raise LookupError("example is not found as a file or directory")
 
+    def run_example(self, name=None, *pytest_args):
```

Comment:
**required** this one seems problematic due to leaving state ehind, as is this api is only oblivious/sensible to core devs

im pretty sure a plugin author would misuse it running it twice

for now i propose using the markers to declare what example to copy to keep apart the folder structure prep and the test running

Reply:
Good point, I can remove it.